code/modules/atmospherics/machinery.dm

```diff
@@ -1,62 +1,52 @@
-/*
-Quick overview:
-
-Pipes combine to form pipelines
-Pipelines and other atmospheric objects combine to form pipe_networks
-	Note: A single pipe_network represents a completely open space
-
-Pipes -> Pipelines
-Pipelines + Other Objects -> Pipe network
-
+ABSTRACT_TYPE(/obj/machinery/atmospherics)
+
+/**
+ * Quick overview:
+ * Pipes combine to form pipelines.
+ * Pipelines and other atmospheric objects combine to form pipe_networks.
+ * Note: A single pipe_network represents a completely open space.
+ *
+ * Pipes -> Pipelines
+ * Pipelines + Other Objects -> Pipe network
```

Comment:
is this block doc comment attached to `ABSTRACT_TYPE(/obj/machinery/atmospherics)`? It looks like it should be but it might be worth double checking.

Reply:
it is not attached, dont want it to be.

Reply:
in that case line 3 can remove one asterisk